### ISSUE TITLE
Unwanted upwards scroll from end of page

### ISSUE THREAD

**Issue by tyler-barnes** [lukehaas/Scrollify#180]
Hello,
Thanks again for this incredible plugin.
If I open a lightbox on the page (prettyPhoto.js) and then close it and then scroll to the bottom of the page - scrollify will automatically scroll back up to the top of the second section (the current/last section). I'm noticing this occurs randomly now and then as well when I'm not opening lightboxes. Is this a bug or is there a setting that needs to be changed? The second section is much longer than the browser height.

**Comment by lukehaas** [comment]
Is scrollify giving you any warning messages in the console?

**Comment by Memes11** [comment]
Same issue here and no messages in console

**Comment by Memes11** [comment]
Just saw that 1.0.7 was out, I updated and this seems solved for me.

**Comment by jhirsch** [comment]
I am experiencing this exact issue with the latest version of Scrollify. My sections are all 100% height, and I have a fixed div on top of everything with animated content powered by ScrollMagic.

Everything is working perfectly so far, except for this one issue.

Thanks.

**Comment by lukehaas** [comment]
@jhirsch do you have a live demo available?

**Comment by jhirsch** [comment]
@lukehaas Thanks for the quick reply. No live demo handy, but I will try and isolate the issue and put something up somewhere. Stay tuned, and thanks again.

**Comment by jhirsch** [comment]
@lukehaas fixed it. total user error. apologies for wasting your time.

BUT - I will tell you the problem in case this ever comes up again, or if you want to watch for this condition in the library ...

`scroll-panel` is the class I assigned as the `section` when initializing scrollify.
I had a div with the `scroll-panel` class contained inside the content div that I had fixed on top of everything to hold the animations. Like so:

```
<div class="scroll-panel" id="panel1"></div>
<div class="scroll-panel" id="panel2"></div>
<div class="scroll-panel" id="panel3"></div>
<div class="scroll-panel" id="panel4"></div>

<div id="main-container">
     <div class="scroll-panel">
    </div>
</div>
```

`main-container` is fixed at 0,0 on top of everything else.
Once I remove the class from that div, everything is cool.

I hope this is helpful. Thanks again.
